underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -39,6 +43,7 @@
 import alluxio.util.UnderFileSystemUtils;
 import alluxio.util.network.NetworkAddressUtils;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
This is unused.

Error:  /usr/src/alluxio/underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java:46:8: Unused import - com.google.common.annotations.VisibleForTesting. [UnusedImports]